CODEOWNERS

```diff
@@ -1,22 +1,31 @@
-# Global owners.
-@ThePrimeagen
+# Global code owner
+*                 @ThePrimeagen
+
+# DevOps
+/.github/         @novusnota
+
+# Language maintainers
+/bash/            @vgel
+/c/               @zivlakmilos
+/cpp/             @nicdgonzalez
+/crystal/         @devnote-dev
+/csharp/          @MarioGK
+/dart/            @shreyassanthu77
+# TODO: Consider uniting those two:
+/elixir-shyryan/  @ryanwinchester
+/elixir/          @ryanwinchester
+/erlang/          @mikimaine
+/fortran/         @greuben92
+/fsharp/          @meenzen
+/java/            @xNaCly
```

Comment:
From my understanding on Stream yesterday I'm supposed to be a Java Code Owner, Prime could confirm.

Reply:
There's a whole list of new people from last stream to put in this file, but yeah